build.gradle

```diff
@@ -69,6 +70,12 @@ dependencies {
     compile "org.modelmapper:modelmapper:$version_modelmapper"
     compile "org.apache.commons:commons-lang3:$version_commons_lang"
     compile "com.google.inject:guice:$version_guice"
+    
+    // apiman deps
+    compile "io.apiman:apiman-gateway-engine-beans:$version_apiman"
+    compile "io.apiman:apiman-common-plugin:$version_apiman"
+    compile "io.apiman:apiman-manager-api-core:$version_apiman"
+    runtime "io.apiman:apiman-distro-data:$version_apiman"
```

Comment:
I was having issues with eclipse + gradle (probably should switch to intellij for good...); probably don't need this as a runtime dep.

`distro-data` contains various useful resources such as a list of inbuilt policies